lib/pkg/types.ts

```diff
@@ -2,7 +2,7 @@ import type { SbSDKOptions } from "@storyblok/js";
 import type { SvelteComponent } from "svelte";
 
 export interface SbSvelteComponentsMap {
-  [name: string]: typeof SvelteComponent;
+  [name: string]: typeof SvelteComponent<any>;
```

Comment:
@roberto-butti if the SvelteComponent type is a Generic Type, better to pass unknown.

Reply:
this is suggested by svelte migration guide. But happy to discuss about it.